packages/embedder/ai_models/TextEmbeddingsInference.ts

```diff
@@ -27,6 +31,28 @@ export class TextEmbeddingsInference extends AbstractEmbeddingsModel {
     super(config)
   }
 
+  async getTokens(content: string) {
+    const fetchOptions = {
+      body: JSON.stringify({inputs: content}),
+      deadline: new Date(new Date().getTime() + MAX_REQUEST_TIME_S * 1000),
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json; charset=utf-8'
+      },
+      method: 'POST'
+    }
+
+    try {
```

Comment:
+2

This is very nice. It's so cool to see that we're riding the edge of the hugging face container development and using these endpoints as they ship them.